tests/functional/testplan/testing/multitest/test_suite_decorators.py

```diff
@@ -100,7 +100,7 @@ def test_basic_multitest():
         assert len(mt_entry.entries) == 2  # 2 Suites
         for st_entry in mt_entry.entries:
             assert isinstance(st_entry, TestGroupReport)
-            assert len(st_entry.entries) == 3  # 3 Testcases
```

Comment:
FYI the reason this is changed... previously if a setup method was defined that didn't take a `result` parameter, no report entry would be generated in the report. I made a small tweak to generate a report object anyway, since the setup could still raise an Exception and it makes sense to store details of that Exception inside the setup report. Plus for consistency, I think it's simpler if a setup method always generates some kind of report. Let me know if you have any questions or concerns.